crowdsec-docs/docs/appsec/intro.md

```diff
@@ -43,7 +43,7 @@ This leads to two possible outcomes:
 
 ### 3 Security Engine: out-of-band rules processing
 
-In the background, the security engine will then evaluate the out-of-band rules. These rules do not impact performance or response time, as they are evaluated after the AppSec component instructs the webserver to continue or stop processing the request.
+In the background, the security engine will then evaluate the out-of-band rules. These rules do not impact performance or response time, as they are evaluated after the AppSec Component instructs the webserver to continue or stop processing the request.
```

Comment:
```
Behind the scenes, the security engine evaluates the out-of-band rules. These rules do not affect performance or response time because they are assessed after the AppSec Component has directed the webserver to either proceed with or halt the request processing.
```